packaging/debian/postinst

```diff
@@ -52,20 +51,38 @@ case "$1" in
 	# TODO: should not download binaries if it's only a reconfigure
 	echo "Downloading binaries..."
 	for binary in "storagenode" "storagenode-updater"; do
-	    wget -O /tmp/${binary}.zip ${BINARIES_SERVER}/${binary}_linux_amd64.zip
-	    unzip /tmp/${binary}.zip -d /var/storagenode/
-	    mv /var/storagenode/${binary}_linux_amd64 /var/storagenode/${binary}
+	    tmpfile=$(mktemp /tmp/${binary}.zip.XXXXXX)
+	    wget -O $tmpfile ${BINARIES_SERVER}/${binary}_linux_amd64.zip
+	    unzip $tmpfile -d /var/lib/storagenode
+	    mv /var/lib/storagenode/${binary}_linux_amd64 /var/lib/storagenode/${binary}
 	    # so that we're in PATH easily
-	    ln -s /var/storagenode/${binary} /usr/bin/${binary}
+	    ln -s /var/lib/storagenode/${binary} /usr/bin/${binary}
```

Comment:
This will fail if the symlink already exists, for example if the package is being upgraded.

You might consider using `dh_link` to create the symlink; `dh_link` will be called by debhelper during package build anyway, so all you'd need to do is create a file called `debian/storagenode.links` containing the one line:

    /var/lib/storagenode/storagenode  /usr/bin

And then similarly for storagenode-updater. That will let the links be managed as owned by the respective packages, and cleaned up when the package is removed, etc.